Revit_oM/Requests/FilterByVisibleInView.cs

```diff
@@ -0,0 +1,40 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2020, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.oM.Data.Requests;
+using System.ComponentModel;
+
+namespace BH.oM.Adapters.Revit.Requests
+{
+    [Description("IRequest that filters all elements visible in a given Revit view.")]
+    public class FilterByVisibleInView : IRequest
```

Comment:
I think request names should always end in `request` for compliance?
Maybe `VisibleInViewRequest`? I think the fact that the request "filters" should be implied in its nature of request. In fact, I think that for the same reason we will rename FilterRequest into TypeTagRequest or something similar.

Reply:
We had a few iterations over that with @tiagogrossi plus a call with @FraserGreenroyd, where we agreed that it makes sense to drop the ambiguous _Request_ suffix in favour of something more human readable. After going back and forth, I ended up with _Filter_ prefix system that I hope to be clear to the end user (before it was not clear even for me 🙊 🙈).